### DISCUSSION TITLE
actions/download-artifact results in Error: Unable to resolve action `actions/download-artifact@4`, unable to find version `4`

### DISCUSSION THREAD

**Discussion by AgostonSzepessy** [community/community#109735] [Actions] [answered]
### Select Topic Area

Question

### Body

I'm trying to create a release pipeline, and as part of it I upload an artifact and then download it. However, the [download step](https://github.com/AgostonSzepessy/wheat/blob/185e8ee79f5d0e444ed44a31db4e369a548a3ca0/.github/workflows/release.yml#L86) keeps resulting in this [error message](https://github.com/AgostonSzepessy/wheat/actions/runs/8024552010/job/21923538569#step:1:25):

```
Error: Unable to resolve action `actions/download-artifact@4`, unable to find version `4`
```

I tried switching to version 3 as well, but I still get the same error (except it says unable to find version 3).

Do I need to give some sort of permission for this action to work in my repo? Or am I missing an argument to the action? This is my first time trying to use actions, so I'm not sure what's going wrong.

**Comment by ChristopherHX** [comment, marked as answer]
You should try `actions/download-artifact@v4`, instead. `v4` is a git tag of the `actions/download-artifact` repository

**Reply by AgostonSzepessy** [reply]
Oh, thank you! I didn't even notice that. That fixed it.
